docs/source/top_level_elements/originInfo_dates.rst

```diff
@@ -0,0 +1,282 @@
+        <dateCreated encoding="w3cdtf" keyDate="yes" point="start">1941</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> dcterms:created "1941" .
+
+
+dateCreated[@keyDate]
+---------------------
+
+The keyDate attribute is always "yes".
+
+.. code-block:: xml
+
+    @keyDate = "yes"
+
+
+dateCreated[@qualifier]
+-----------------------
+
+The qualifier attribute has three distinct values: "inferred", "approximate", and "questionable". It usually appears with other attributes but not always; e.g. `ekcd:95 <https://digital.lib.utk.edu/collections/islandora/object/ekcd:95/datastream/MODS>`_:
+
+.. code-block:: xml
+
+    <originInfo>
+        <dateCreated qualifier="inferred">1955</dateCreated>
+        <dateCreated encoding="edtf" keyDate="yes">1955</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> dcterms:created "1955", "1955~" .
+
+.. code-block:: xml
+
+    <originInfo>
+        <dateCreated>Undated</dateCreated>
+        <dateCreated encoding="edtf" keyDate="yes" point="start" qualifier="inferred">1910</dateCreated>
+        <dateCreated encoding="edtf" keyDate="yes" point="end" qualifier="inferred">1955</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> dcterms:created "Undated", "1910~/1955~" .
+
+dateCreated[@point]
+-------------------
+
+The point attribute has two distinct values: "start" and "end". They are frequently, but not consistently paired; e.g. `volvoices:2152 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A2152/datastream/MODS>`_: and `volvoices:3849 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A3849/datastream/MODS>`_:
+
+.. code-block:: xml
+
+    <originInfo>
+        <dateCreated>1915</dateCreated>
+        <dateCreated encoding="edtf" keyDate="yes" point="start">1915</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> dcterms:created "1915", "1915/" .
+
+.. code-block:: xml
+
+    <originInfo>
+        <dateCreated>approximately between 1940 and 1950</dateCreated>
+        <dateCreated encoding="edtf" keyDate="yes" point="start" qualifier="approximate">1940</dateCreated>
+        <dateCreated encoding="edtf" keyDate="yes" point="end">1950</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> dcterms:created "approximately between 1940 and 1950", "1940~/1950" .
+
+dateIssued
+----------
+
+`dateIssued`'s attributes and possible values follow the examples in `dateCreated` very closely. The primary difference between the two is that `dateIssued` is used in records describing serials.
```

Comment:
Not super important, but dateIssued is broadly used to describe published materials (not just serials, though for us this is the vast majority) - https://www.loc.gov/standards/mods/userguide/origininfo.html#dateissued The sheet music collection is a good example of a non-serial collection that uses dateIssued.

Reply:
Thanks for catching this. I updated the descriptive text - let me know if it needs additional tweaking.